<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-12-30T11:15:15Z" name="Default suite" finished-at="2020-12-30T11:15:27Z" duration-ms="11618">
    <groups>
    </groups>
    <test started-at="2020-12-30T11:15:15Z" name="Default test" finished-at="2020-12-30T11:15:27Z" duration-ms="11618">
      <class name="DDFPack.DDFProj.tests.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:DDFPack.DDFProj.tests.LoginTest@79e4c792]" started-at="2020-12-30T11:15:15Z" name="init" finished-at="2020-12-30T11:15:15Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="doLoginTest(java.util.HashMap)[pri:0, instance:DDFPack.DDFProj.tests.LoginTest@79e4c792]" started-at="2020-12-30T11:15:16Z" name="doLoginTest" data-provider="getData" finished-at="2020-12-30T11:15:25Z" duration-ms="8884" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, ExpectedResult=Success, Username=seleniumbyarun99@gmail.com, Browser=Firefox, Password=Demopassword$}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.click()" because "element" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.click()" because "element" is null
at DDFPack.DDFProj.base.BaseTest.click(BaseTest.java:113)
at DDFPack.DDFProj.base.BaseTest.doLogin(BaseTest.java:94)
at DDFPack.DDFProj.tests.LoginTest.doLoginTest(LoginTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLoginTest -->
        <test-method is-config="true" signature="testClosure()[pri:0, instance:DDFPack.DDFProj.tests.LoginTest@79e4c792]" started-at="2020-12-30T11:15:25Z" name="testClosure" finished-at="2020-12-30T11:15:27Z" duration-ms="2083" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testClosure -->
        <test-method signature="doLoginTest(java.util.HashMap)[pri:0, instance:DDFPack.DDFProj.tests.LoginTest@79e4c792]" started-at="2020-12-30T11:15:27Z" name="doLoginTest" data-provider="getData" finished-at="2020-12-30T11:15:27Z" duration-ms="7" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=N, ExpectedResult=Success, Username=seleniumbyarun99@gmail.com, Browser=Chrome, Password=Demopassword$}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test as Runmode is set to N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test as Runmode is set to N
at DDFPack.DDFProj.tests.LoginTest.doLoginTest(LoginTest.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLoginTest -->
        <test-method is-config="true" signature="testClosure()[pri:0, instance:DDFPack.DDFProj.tests.LoginTest@79e4c792]" started-at="2020-12-30T11:15:27Z" name="testClosure" finished-at="2020-12-30T11:15:27Z" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testClosure -->
      </class> <!-- DDFPack.DDFProj.tests.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
